### ISSUE TITLE
Version 12 compability?

### ISSUE THREAD

**Issue by Eireman** [DFreds/dfreds-convenient-effects#328]
Foundry is currently in a public testing v.4 beta and I have asked to NOT be part of the testing as my server runs live streamed games.

Is there any time table for this module to inherit v12 verbosity?

**Comment by Eireman** [comment]
v12.325 is released, and still no response?

**Comment by vaka42** [comment]
Upvote :D  Nice Addon, must have for all :)

**Comment by tslays** [comment]
+1 on V12. No ETA yet?

**Comment by Lector** [comment]
Also an upvote from me.

**Comment by BenNelsonNeb** [comment]
Another upvote!

**Comment by brennanmuir** [comment]
Needed for Plutonium to work

**Comment by BenNelsonNeb** [comment]
> Needed for Plutonium to work

If you uninstall the "automation addon companion module" you can get Plutonium to work...but there are a LOT of modules that rely on Dfreds.

**Comment by SagaTympana** [comment]
Nobody cares what’s required for your piracy module to work.

**Comment by EvldoaTV** [comment]
Another Upvote amazing Mod needs update badly

**Comment by VirusNik21** [comment]
And one another upvote

**Comment by uberDoward** [comment]
Add another vote

**Comment by Geektasticdad** [comment]
Another up vote 👍

**Comment by GomatiGit** [comment]
+1 for an updated version for FoundryVTT v12 stable

**Comment by twagorilla** [comment]
Another upvote here

**Comment by TheTableKnight** [comment]
Can sacrifice an upvote on here for this

**Comment by caspaas** [comment]
> Nobody cares what’s required for your piracy module to work.

the f is wrong with you ? xD how is Dfred piracy

**Comment by thatlonelybugbear** [comment]
> > Nobody cares what’s required for your piracy module to work.
> 
> the f is wrong with you ? xD how is Dfred piracy

Ehmm, the modules mentioned in the messages that the user has replied to, is indeed dubious to say the least. Nobody said that for DFreds. 

Peace ☮️

**Comment by Gacky2k** [comment]
Yet another upvote here from me.  DFred's is practically a required component to help make management of status effects soooo much easier!

**Comment by SirAron111** [comment]
Upvote here as well. DFreds is important. Best effect module to use for its job.

**Comment by thatlonelybugbear** [comment]
FYI. 

It is being worked on, but there is no ETA at this time.

**Comment by Dan69UwU** [comment]
Another upvote!

**Comment by AlexMeuer** [comment]
I've forked this repo and updated the manifest to make it loadable with v12. I haven't made any code changes, so use it at your own risk :shrug:
- https://github.com/AlexMeuer/dfreds-convenient-effects/releases/tag/v6.0.4
- `https://github.com/AlexMeuer/dfreds-convenient-effects/releases/download/v6.0.4/module.json`

**Comment by BenNelsonNeb** [comment]
> I've forked this repo and updated the manifest to make it loadable with v12. I haven't made any code changes, so use it at your own risk 🤷
> 
>     * https://github.com/AlexMeuer/dfreds-convenient-effects/releases/tag/v6.0.4
> 
>     * `https://github.com/AlexMeuer/dfreds-convenient-effects/releases/download/v6.0.4/module.json`

Tried it out...womp womp...no go. Icon shows but does nothing. 
Error: An error occurred while rendering ConvenientEffectsApp 35. this._all is undefined

**Comment by AlexMeuer** [comment]
For me it was enough to get foundry to allow me to enable/disable other modules. Previously, foundry wouldn't let me change any modules since something that depended on convenient effects was enabled and couldn't be unselected so this fixes it for me :sweat_smile:

**Comment by AlexMeuer** [comment]
> FYI.
> 
> It is being worked on, but there is no ETA at this time.

Is that work happening on the `module_rewrite` branch or somewhere else?

**Comment by Sonorpearl** [comment]
> FYI.
> 
> It is being worked on, but there is no ETA at this time.

Thanks for working on it. :-)

**Comment by Fridan99** [comment]
any news about dfreds convenient effect v12? :P

**Comment by Nedrapter** [comment]
Commenting on this to get notified when it's closed, so I can upgrade to v12 :)

**Comment by Trinistar** [comment]
dito

**Comment by obfuscate** [comment]
Commenting on this to get notified when it's closed, so I can upgrade to v12 :)

**Comment by franckflo** [comment]
Thanks a lot for your hard work DFreds, looking forward to using Convenient Effects again with v12.

**Comment by mcabel** [comment]
upvoting this

**Comment by ChattyConnor** [comment]
Excited and grateful for updates! :)

**Comment by Amd321** [comment]
Upvote/notified when ready. Can't DM nearly as easily without this module. It's my favourite one! Thanks, and looking forward to it being updated!

**Comment by carolinex** [comment]
Following

**Comment by wieli99** [comment]
Guys if you want to follow this issue, please use the subscribe button at the top right. No need to announce to everyone!

**Comment by carolinex** [comment]
> Guys if you want to follow this issue, please use the subscribe button at the top right. No need to announce to everyone!

I understand the annoyance, but as far as I know, you only have the option to receive every update on the repo, or just the ones you are participating in. Hence the need to post on the thread you want to follow. If you know of any other way, I'll up to learning!

**Comment by wieli99** [comment]
> > Guys if you want to follow this issue, please use the subscribe button at the top right. No need to announce to everyone!
> 
> I understand the annoyance, but as far as I know, you only have the option to receive every update on the repo, or just the ones you are participating in. Hence the need to post on the thread you want to follow. If you know of any other way, I'm here to learn!


You can enable notifications for a single thread at the top right, without any other interaction.

![image](https://github.com/user-attachments/assets/551f6ce3-07dc-4e23-973d-45af634280b6)

**Comment by carolinex** [comment]
Ugh, I swear I looked everywhere on the mobile interface and couldn't find it, but here it is, at the very bottom of the page. Thanks.

**Comment by TheXeno30** [comment]
> FYI.
> 
> It is being worked on, but there is no ETA at this time.

Is there an ETA for this now?
Thanks for a great module - Players can't live withoout it :)

**Comment by TrofimSelbs** [comment]
For everyone waiting — I found this YouTube video with a workaround to get it to work on v12 with dnd 5e 3.1.2. Hope it works for you!

https://www.youtube.com/watch?v=dQw4w9WgXcQ

**Comment by TheXeno30** [comment]
> For everyone waiting — I found this YouTube video with a workaround to get it to work on v12 with dnd 5e 3.1.2. Hope it works for you!
> 
> https://www.youtube.com/watch?v=dQw4w9WgXcQ

Cruel, but funny!! 🤣🤣🤣

**Comment by macskay** [comment]
> For everyone waiting — I found this YouTube video with a workaround to get it to work on v12 with dnd 5e 3.1.2. Hope it works for you!
> 
> https://www.youtube.com/watch?v=dQw4w9WgXcQ

Fully expected it and still clicked it because of desperation while waiting

**Comment by yyzitai** [comment]
https://discord.gg/Wq8AEV9bWb

DFreds — 2024/08/01 04:45
I am hoping that CE will be v12 compatible in the next week or so. Dear god don't quote me on that

DFreds — 2024/08/01 05:02
If anyone wants to be a Tester, let me know. About to drop a new build to test 😤

top top top top top

**Comment by fegarza7** [comment]
Waiting on this too!!

**Comment by DFreds** [comment]
Compatible in v7.0.0

**Comment by Captain-P-Goldfish** [comment]
just tried with version 12 build 331 and I just get an empty list:
![Unbenannt](https://github.com/user-attachments/assets/2feb236e-44cd-4208-b620-26225c0b076a)
how to fix that?

**Comment by thatlonelybugbear** [comment]
Click on View Backups and Reset

![image](https://github.com/user-attachments/assets/a0741a1f-0b10-49e5-a864-ac4b619a7520)

**Comment by Captain-P-Goldfish** [comment]
tried it twice now but it does not work. List is still empty after the page reloads

**Comment by DFreds** [comment]
I'd log a separate issue. Not sure why the reset wouldn't work 🤔
